### PR TITLE
Add PTRACE_GETREGSET and PTRACE_SETREGSET which are missing on android

### PR DESCRIPTION
**Pull request by s1341**

### CODE REVIEW

File: src/unix/linux_like/android/mod.rs [hunk 1]
```diff
@@ -1277,6 +1277,8 @@ pub const PTRACE_SETOPTIONS: ::c_int = 0x4200;
 pub const PTRACE_GETEVENTMSG: ::c_int = 0x4201;
 pub const PTRACE_GETSIGINFO: ::c_int = 0x4202;
 pub const PTRACE_SETSIGINFO: ::c_int = 0x4203;
+pub const PTRACE_GETREGSET: ::c_uint = 0x4204;
+pub const PTRACE_SETREGSET: ::c_uint = 0x4205;
```
Comment (JohnTitor) [inline src/unix/linux_like/android/mod.rs:1281]: We've declared other `PTRACE_*`s as `c_int`, they may cause a confusion.

Comment (s1341) [reply src/unix/linux_like/android/mod.rs]: Woops. Copy paste error. Fixed.

**Comment by JohnTitor** [comment]
Could you squash commits into one?

**Comment by JohnTitor** [comment]
Merged via e02b7e5c1d894c53d206a7c348f00d32fd995685. Thanks!

**Comment by s1341** [comment]
Thanks! Sorry. Didn't get to the squash

**Pull request closed without merging**